scripts/templater.py

```diff
@@ -117,13 +117,17 @@ def set_template(argv):
 
     name = f"{inspect.stack()[0][3]}"
     log = cli_helpers.setup_logging(user_args, log_name=name)
+    cli_arg = ""
 
     log.info(f"""Running {name} with args: """)
     log.info(f"""{('-' * 70)}""")
     log.info(f"""{('-' * 70)}""")
     for name, val in user_args.__dict__.items():
         if name not in ["config"]:
             log.info("{name:>15s}: {val}".format(name=name, val=val))
+    for arg in argv:
+        cli_arg = cli_arg + " " + arg
+    log.info(f"""re-run settings: templater.py {cli_arg}""")
```

Comment:
This is _exactly_ what the developer was looking for! Nice.

I'll leave a more advance Python trick to consider (there's always more than one way to come at a problem)

Since `argv` is a list, we can use the `join` string method to help gather the contents into a string:

```
log.info(f"""re-run settings: templater.py {' '.join(argv)}""")
```

I really appreciate that you took advantage of `argv` directly here since you had access to it. I would have likely aimed at iterating through the `user_args` dict on my first pass and made it more difficult than it needed to be.

Reply:
Iterating through user_args is exactly what we tried at first too before having that ah-ha moment :)

Reply:
Nice work. +1 to @christinaholtNOAA 's suggestion to use `join()`.

This code has changed slightly on my `condev` branch (which I hope to PR next week, and maybe schedule a quick demo/tour Meet for, since it touches a lot of code), but I will make sure that this feature gets applied correctly there.

BTW, does anybody know the rationale for the (pre-this-work) triple-double-quotes, i.e. `f"""Running {name} with args: """` instead of just `f"Running {name} with args: "`? I've simplified on my branch, to no apparent negative effect, but am curious.

Reply:
Paul, I think it was originally implemented as a single multi-line log message before it was turned into multiple single-line log messages (from this [PR](https://github.com/ufs-community/workflow-tools/pull/203)). We can clean that up on Wednesday.

Reply:
@elcarpenter That makes sense, thanks.